packages/paste-core/components/badge/stories/index.stories.tsx

```diff
@@ -20,16 +20,16 @@ const Wrapper = styled.div(
     display: 'flex',
     columnGap: 'space60',
     alignItems: 'center',
+    rowGap: 'space60',
+    flexWrap: 'wrap',
+    marginBottom: 'space80',
   })
 );
 
 export const AllBadges: React.FC = () => (
   <Wrapper>
-    <Badge as="span" variant="default">
-      Default
-    </Badge>
-    <Badge as="span" variant="info">
-      Info
+    <Badge as="span" variant="neutral">
```

Comment:
suggestion(blocker): we should be keeping default and info variant stories for regression testing. Even though we don't recommend using them, we do support them, and we should be protecting our customers from regressions that might happen. Keeping the stories allows us to make sure nothing happens to them when we're working on badge

Reply:
Ah I see, that's a great point. I'll add the outdated variants back in. Thank you!